### DISCUSSION TITLE
Are `notFound` and similar utilities deprecated in version 7?

### DISCUSSION THREAD

**Discussion by RobertGemmaJr** [sergiodxa/remix-utils#265] [Q&A] [answered]
`import { notFound } from "remix-utils";` is throwing an error after updating to version 7 of `remix-utils`. I know I need to import it from some nested path but I can't seem to find where it exists, nor are they on the documentation.

Have these utilities been deprecated? If not, how do I import them correctly?

**Comment by sergiodxa** [comment, marked as answer]
Yes, they were just wrappers of `json` adding a specific status code so I removed it.

**Reply by RobertGemmaJr** [reply]
Could you add that to the release notes maybe? It wasn't at all clear what happened to them
